test/parallel/test-internal-dtrace.js

```diff
@@ -0,0 +1,24 @@
+// Flags: --expose-internals
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+const {
+  DTRACE_HTTP_CLIENT_REQUEST,
+  DTRACE_HTTP_CLIENT_RESPONSE,
+  DTRACE_HTTP_SERVER_REQUEST,
+  DTRACE_HTTP_SERVER_RESPONSE,
+  DTRACE_NET_SERVER_CONNECTION,
+  DTRACE_NET_STREAM_END
+} = require('internal/dtrace');
+
+// We're just testing to make sure these are always defined and
```

Comment:
Why is this necessary? I am pretty sure our test suite already covers all of these call sites and if those functions are not callable, the test would fail because of unexpected errors?

Reply:
Just making them more visible really. The additional test doesn't hurt anything.